### ISSUE TITLE
Accessibility of Modals

### ISSUE THREAD

**Issue by estelle** [gitpod-io/website#209]
per a11y, modal close buttons should have focus when the modal is opened, and when the popup is closed, focus should return to whatever opened the popup. I did not work on that.

This impact the careers page.
